lenovo/yoga/6/13ALC6/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, pkgs, ...  }:
+
+{
+  imports = [
+    ../../../thinkpad/yoga.nix
+    ../../../../common/gpu/amd/default.nix
+  ];
+
+  boot.initrd.kernelModules = [ "ideapad_laptop" ];
+  hardware.opengl.extraPackages = with pkgs; [
+    vaapiVdpau
+    libvdpau-va-gl
+  ];
+
+  # Latest Kernel
+  boot.kernelPackages = lib.mkDefault pkgs.linuxPackages_latest;
+
+  # energy savings
+  hardware.bluetooth.powerOnBoot = lib.mkDefault false;
+  services.power-profiles-daemon.enable = false;
+  services.tlp.enable = lib.mkDefault true;
+  boot.kernelParams = ["mem_sleep_default=deep" "pcie_aspm.policy=powersupersave"];
+  powerManagement.enable = lib.mkDefault true;
+  powerManagement.cpuFreqGovernor = lib.mkDefault "ondemand";
+  services.tlp.settings = {
+    # CPU
+    CPU_SCALING_GOVERNOR_ON_BAT="ondemand";
```

Comment:
Is schedutil not the new gold standard for AMD or is this something that only exists for newer generations?

Reply:
I wouldn't mind if schedutil saves energy better

Reply:
But I don't have time to test such things as the current configuration is best for getting 10 hours of battery easily

Reply:
Ok. than please drop all tlp settings from now and leave them in your own configuration. It is otherwise to hard to maintain a configuration where we don't know why certain setting where chosen.